src/dos/dos.cpp

```diff
@@ -1204,6 +1204,50 @@ static Bitu DOS_26Handler(void) {
     return CBRET_NONE;
 }
 
+DOS_Version DOS_ParseVersion(const char *word, const char *args)
+{
+    struct DOS_Version new_version;
```

Comment:
Indent this line using tab instead of spaces.

Also, this is C++, not C - please drop `struct` keyword :)

Reply:
Nothing more from me @dreamer; everything I noted has been addressed.

Reply:
Done.